CFC_WebApp/tests/TestFilterTrips.py

```diff
@@ -0,0 +1,99 @@
+import unittest
+import json
+from utils import load_database_json, purge_database_json
+from main import tripManager
+from pymongo import MongoClient
+import logging
+from get_database import get_db, get_mode_db, get_section_db, get_trip_db
+import re
+# Needed to modify the pythonpath
+import sys
+import os
+from datetime import datetime, timedelta
+from dao.user import User
+from dao.client import Client
+import tests.common
+
+logging.basicConfig(level=logging.DEBUG)
+
+class TestFilterTrips(unittest.TestCase):
```

Comment:
I just thought that there was a lot of functionality already being tested in the TripManager so moving the testing of this feature to a different file would help seperate the testing functionality. If we add an __init__.py file in the test directory then we can avoid importing in each testing file.